packages/react-server/src/ReactFizzServer.js

```diff
@@ -520,6 +521,94 @@ function renderSuspenseBoundary(
   popComponentStackInDEV(task);
 }
 
+function renderSuspenseBoundaryWithoutFallback(
```

Comment:
I'd call this renderBackupSuspenseBoundary instead since that's what we refer to them as now even though we haven't updated the api yet. "WithoutFallback" sounds more like `fallback={undefined}`.